front/src/Api/Events/ui/UIWebsite.ts

```diff
@@ -40,8 +40,8 @@ export type UIWebsiteSize = z.infer<typeof isUIWebsiteSize>;
 
 export const isCreateUIWebsiteEvent = z.object({
     url: z.string(),
-    visible: z.optional(z.boolean()),
-    allowApi: z.optional(z.boolean()),
+    visible: z.boolean().optional(),
+    allowApi: z.boolean().optional(),
```

Comment:
Why this update and why doesn't apply on all `z.optional(...)`?

Reply:
The optional wrapper is the legacy syntax and will be deleted in future.